e2etest/layer2_test.go

```diff
@@ -28,6 +29,7 @@ import (
 	"github.com/mikioh/ipaddr"
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
```

Comment:
I think you should not use dot here like the other file: https://github.com/metallb/metallb/blob/eb3c241b571f6838843002bedada5aee28ce14b5/e2etest/e2etest_suite_test.go#L32

Reply:
Will change

Reply:
Done